src/mtype.d

```diff
@@ -9497,27 +9453,69 @@ public:
         return params;
     }
 
+    extern (D):
```

Comment:
Looking at mtype.h, it would never have worked even before this PR. I just deleted the extern(D) declarations from it.